driver/mysql.go

```diff
@@ -0,0 +1,19 @@
+package driver
```

Comment:
no need to create new package just to create db object,  do that in main.go

Reply:
I also done testing for database connection. That's why wrote it in another file.

Reply:
no need to test db connection.

Reply:
ok